pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -135,6 +135,23 @@ type VitessClusterUpdateStrategy struct {
 	// Default: External
 	// +kubebuilder:validation:Enum=External,Immediate
 	Type *VitessClusterUpdateStrategyType `json:"type,omitempty"`
+
+	// DataVolumeClaimResize selects the strategy chosen when updating the underlying storage
+	// value in the DataVolumeClaimTemplate for a tablet pool's persistent volumes.
+	//
+	// Supported options are:
+	//
+	// - External: Schedule disk size updates on tablet pools and persistent
+	//   volume claims but wait for an external tool to release them by adding
+	//   the 'rollout.planetscale.com/released' annotation.
+	// - Immediate: Release updates to all tablet pools and persistent volume
+	//   claims as soon as the VitessCluster spec is changed. Perform rolling
+	//   restart of one tablet Pod per shard at a time, with automatic planned
+	//   reparents whenever possible to avoid master downtime.
+	//
+	// Default: External
```

Comment:
Need to make this actually happen in `DefaultUpdateStrategy()`.

Reply:
Done.